### PR TITLE
Update @zeit/ncc to @vercel/ncc

### PR DESCRIPTION
**Pull request by nikeee**
**Description:**
Updates some dependencies. Also, `@zeit/ncc` is no longer maintained and moved to `@vercel/ncc`.

**Check list:**
- [x] Mark if documentation changes are required.
- [x] Mark if tests were added or updated to cover the changes.

### CODE REVIEW

**Comment by vsafonkin** [comment]
Hi @nikeee, sorry for the delay, could you please update your PR from `main` and resolve conflicts?

**Comment by nikeee** [comment]
@vsafonkin done

File: package-lock.json [hunk 1]
```diff
@@ -1,7959 +1,8036 @@
...
-      "license": "MIT",
-      "dependencies": {
-        "@actions/core": "^1.6.0",
-        "@actions/http-client": "^1.0.6",
-        "@actions/io": "^1.0.2",
-        "@actions/tool-cache": "^1.5.5",
-        "semver": "^6.1.1"
-      },
-      "devDependencies": {
-        "@types/jest": "^27.0.2",
```
Comment (brcrista) [inline package-lock.json:19]: @nikeee looks like the line endings on this file changed to `CRLF`.  Can you change it back to `LF`?

@vsafonkin we probably should add `* text=auto` to https://github.com/actions/setup-go/blob/main/.gitattributes

**Comment by brcrista** [comment]
@nikeee the `check-dist` run is failing -- the `dist` code (which is what actually runs) needs to be updated since package versions changed.

You can just download the artifact from that run and check it in: https://github.com/actions/setup-go/runs/5814071208

**Comment by dmitry-shibanov** [comment]
Hello @nikeee. I'm going to close your pull request in favour of this one. https://github.com/actions/setup-go/pull/229

**Pull request closed without merging**